packages/components/src/spinners/SpinnerOverlay.test.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import { renderWithTheme } from '../internal/test-utils';
+import { SpinnerOverlay, SpinnerOverlayMode } from './SpinnerOverlay';
+
+describe('SpinnerOverlay', () => {
+  it('renders a spinner icon', () => {
+    // Act
+    renderWithTheme(<SpinnerOverlay />);
+
+    // Assert
+    const icon = document.querySelector('[data-icon="loadingSpinner"]');
+    expect(icon).toBeInTheDocument();
+  });
+  it.each(['Absolute', 'Fixed', 'Top', 'Transparent', 'FullScreen', 'Default'] satisfies SpinnerOverlayMode[])(
+    'renders mode',
```

Comment:
Nit: it would be nice to include the mode in the test description (which you can do by referencing the variable).

Reply:
implemented